ast/ddl.go

```diff
@@ -667,15 +668,16 @@ const (
 type AlterTableSpec struct {
 	node
 
-	Tp            AlterTableType
-	Name          string
-	Constraint    *Constraint
-	Options       []*TableOption
-	NewTable      *TableName
-	NewColumn     *ColumnDef
-	OldColumnName *ColumnName
-	Position      *ColumnPosition
-	LockType      LockType
+	Tp              AlterTableType
+	Name            string
+	Constraint      *Constraint
+	Options         []*TableOption
+	NewTable        *TableName
+	NewColumn       *ColumnDef
+	AppendedColumns []*ColumnDef
```

Comment:
1. Refer to MySQL alter syntax,  it add multiple columns without positions. 
`alter_specification:
    table_options
  | ADD [COLUMN] col_name column_definition
        [FIRST | AFTER col_name ]
  | ADD [COLUMN] (col_name column_definition,...)`

2.This field named 'NewColumn' is used by modification and added action. In the syntax, it's wired that one OldColumnName to a slice of NewColumn.

@hanfei1991, waiting for your suggestion.